backend/api/Areas/Project/Mapping/Dispose/ProjectMap.cs

```diff
@@ -181,7 +181,7 @@ public void Register(TypeAdapterConfig config)
                         ExemptionApprovedOn = src.ExemptionApprovedOn,
                         DisposedOn = src.DisposedOn,
                         SalesCost = src.SalesCost,
-                        NetProceeds = src.NetProceeds,
```

Comment:
This code was just getting the net proceeds from the database, instead of doing the calculation.